mono/metadata/exception.c

```
@@ -168,9 +168,19 @@ MonoException *
 mono_exception_from_name_two_strings (MonoImage *image, const char *name_space,
 				      const char *name, MonoString *a1, MonoString *a2)
 {
-	MonoClass *klass = mono_class_load_from_name (image, name_space, name);
+	MonoError error;
+	MonoClass *klass;
+	MonoException *ret;
 
-	return create_exception_two_strings (klass, a1, a2);
+	klass = mono_class_load_from_name (image, name_space, name);
+
+	ret = create_exception_two_strings (klass, a1, a2, &error);
+	if (!is_ok (&error)) {
+		mono_error_cleanup (&error);
```

Why swallow the error and return NULL here?